lighthouse-core/audits/installable-manifest.js

```diff
@@ -5,38 +5,127 @@
  */
 'use strict';
 
-const MultiCheckAudit = require('./multi-check-audit.js');
-const ManifestValues = require('../computed/manifest-values.js');
+const Audit = require('./audit.js');
 const i18n = require('../lib/i18n/i18n.js');
+const ManifestValues = require('../computed/manifest-values.js');
 
 const UIStrings = {
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is installable. */
-  title: 'Web app manifest meets the installability requirements',
+  'title': 'Web app manifest and service worker meet the installability requirements',
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is not installable. */
-  failureTitle: 'Web app manifest does not meet the installability requirements',
+  'failureTitle': 'Web app manifest or service worker do not meet the installability requirements',
   /** Description of a Lighthouse audit that tells the user why installability is important for webapps. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Browsers can proactively prompt users to add your app to their homescreen, ' +
-    'which can lead to higher engagement. ' +
+  'description': 'Service worker is the technology that enables your app to use many Progressive ' +
+    'Web App features, such as offline, add to homescreen, and push notifications. ' +
+    'With proper Service Worker and manifest implementations, browsers can proactively prompt ' +
+    'users to add your app to their homescreen, which can lead to higher engagement. ' +
     '[Learn more](https://web.dev/installable-manifest/).',
+  /** Description Table column header for the observed value of the Installability Error statistic. */
+  'columnValue': 'Installability Error',
+  /**
+   * @description [ICU Syntax] Label for an audit identifying the number of installability errors found in the page.
+  */
+  'displayValue': `{itemCount, plural,
+    =1 {1 error}
+    other {# errors}
+    }`,
+  /**
+   * @description Error message describing a DevTools error id that was found and has not been identified by this audit.
+   * @example {platform-not-supported-on-android} errorId
+   */
+  'noErrorId': `Installability error id '{errorId}'`,
+  /** Error message explaining that the page is not loaded in the frame.  */
+  'not-in-main-frame': 'Page is not loaded in the main frame',
```

Comment:
Is there a reason the error message strings' keys don't use camelCase like most UIStrings keys?

Reply:
Ah, nevermind. Seems to be so that the mapping should be the same as the devtools protocol's.